Pod/Classes/VOKCoreDataManager.m

```diff
@@ -444,18 +445,18 @@ - (BOOL)deleteAllObjectsOfClass:(Class)managedObjectClass
 {
     contextOrNil = [self safeContext:contextOrNil];
     NSFetchRequest *fetchRequest = [self fetchRequestWithClass:managedObjectClass predicate:predicate];
-    [fetchRequest setIncludesPropertyValues:NO];
+    fetchRequest.includesPropertyValues = NO;
     
     NSError *error;
     NSArray *results = [contextOrNil executeFetchRequest:fetchRequest error:&error];
     if (!results) {
-        VOK_CDLog(@"%s Fetch Request Error\n%@", __PRETTY_FUNCTION__, [error localizedDescription]);
+        VOK_CDLog(@"Fetch Request Error\n%@", [error localizedDescription]);
         return NO;
     }
     
-    [results enumerateObjectsUsingBlock:^(id obj, NSUInteger idx, BOOL *stop) {
-        [contextOrNil deleteObject:obj];
-    }];
+    for (id object in results) {
```

Comment:
`object` could be noted as an `NSManagedObject` instead of `id` but it doesn't really matter